packages/sampler/stories/qa/VirtualGridList.js

```diff
@@ -2,17 +2,23 @@ import {VirtualGridList, VirtualListBase} from '@enact/moonstone/VirtualList';
 import {VirtualListBase as UiVirtualListBase} from '@enact/ui/VirtualList/VirtualListBase';
 import GridListImageItem from '@enact/moonstone/GridListImageItem';
 import ri from '@enact/ui/resolution';
+import {ScrollableBase as UiScrollableBase} from '@enact/ui/Scrollable';
 import React from 'react';
 import {storiesOf} from '@storybook/react';
 import {action} from '@storybook/addon-actions';
 
 import {boolean, number, select} from '../../src/enact-knobs';
 import {mergeComponentMetadata} from '../../src/utils/propTables';
 
-const Config = mergeComponentMetadata('VirtualGridList', VirtualGridList, VirtualListBase, UiVirtualListBase);
+const Config = mergeComponentMetadata('VirtualList', UiVirtualListBase, UiScrollableBase, VirtualListBase);
 
 const
 	defaultDataSize = 1000,
+	wrapOption = {
+		'false': false,
```

Comment:
Any reason to use `'false'` as a property name instead of just `false`?

Reply:
The data type of result of `select` function is String.

Reply:
I've got your intention. I'll try to fix them and test them.